tests/SprykerEcoTest/Zed/Adyen/Business/BaseSetUpTest.php

```diff
@@ -436,6 +441,22 @@ protected function createRedirectResponseTransfer(OrderTransfer $orderTransfer):
         return $redirectResponseTransfer;
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     * @return \Generated\Shared\Transfer\AdyenRedirectResponseTransfer
+     */
+    protected function createRefusedRedirectResponseTransfer(OrderTransfer $orderTransfer): AdyenRedirectResponseTransfer
```

Comment:
this method duplicates code from `createRedirectResponseTransfer`
result code could be passed as an argument instead